### ISSUE TITLE
Noticable input lag in Metroid Prime Pinball 

### ISSUE THREAD

**Issue by kurayami6** [StapleButter/melonDS#348]
I'm not sure if it's a problem specific to that game, but it's very noticeable when working the flippers. Especially when you've got multiple balls going and have to press the buttons really quickly.

**Comment by iamthelaw2041** [comment]
Yeah this doesn't seem right at all. I might be imagining it but it feels as if the flippers get less responsive the longer the round goes on. 

![Capture](https://user-images.githubusercontent.com/94656803/144499305-a3bfbb5c-c9d1-400a-a298-e9f0678b069d.PNG)
